lib/jnpr/jsnapy/jsnapy.py

```diff
@@ -1067,7 +1081,7 @@ def check_arguments(self):
                     None)
                 sys.exit(1)
             else:
-                if self.args.file is None:
+                if self.args.file is None and (self.args.test_file is None or self.args.hostname is None or self.args.login is None or self.args.passwd is None):
```

Comment:
You code violates pep8 (line not more than 80 char, can we indent code properly.